nixos/modules/programs/_1password.nix

```diff
@@ -8,19 +8,16 @@ let
 
 in
 {
+  imports = [
+    (mkRemovedOptionModule [ "programs" "_1password" "gid" ] ''
+      A preallocated GID would be used instead.
```

Comment:
Why "would" and not "will"? Is this hypothetical or conditional in any way?

Reply:
See https://github.com/NixOS/nixpkgs/pull/172058#discussion_r873168425